src/styles/block/card.scss

```diff
@@ -0,0 +1,120 @@
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+body {
+  padding: 32px 16px 16px;
+  width: 200px;
+  margin: 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
```

Comment:
you can reset it all tags you need (in the index file or even better in reset.scss)

```
h2, p {
  margin: 0;
}
```